src/test/java/uk/gov/hmcts/reform/migration/service/migrate/MultipleHearingsOutcomeMigrationTest.java

```diff
@@ -0,0 +1,191 @@
+package uk.gov.hmcts.reform.migration.service.migrate;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.domain.hmc.CaseHearing;
+import uk.gov.hmcts.reform.domain.hmc.HearingDaySchedule;
+import uk.gov.hmcts.reform.domain.hmc.HearingsGetResponse;
+import uk.gov.hmcts.reform.migration.hmc.HmcHearingsApiService;
+import uk.gov.hmcts.reform.migration.service.HearingOutcomeService;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CaseOutcome;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Hearing;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOutcome;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOutcomeDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingRoute;
+import uk.gov.hmcts.reform.sscs.ccd.domain.JointParty;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SchedulingAndListingFields;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsDeprecatedFields;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsFinalDecisionCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsPipCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Venue;
+import uk.gov.hmcts.reform.sscs.ccd.domain.YesNo;
+import uk.gov.hmcts.reform.sscs.reference.data.model.HearingChannel;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.migration.service.migrate.CompletedHearingsOutcomesMigration.CASE_OUTCOME_MIGRATION_DESCRIPTION;
+import static uk.gov.hmcts.reform.migration.service.migrate.CompletedHearingsOutcomesMigration.CASE_OUTCOME_MIGRATION_ID;
+import static uk.gov.hmcts.reform.migration.service.migrate.CompletedHearingsOutcomesMigration.CASE_OUTCOME_MIGRATION_SUMMARY;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.HearingRoute.LIST_ASSIST;
+import static uk.gov.hmcts.reform.sscs.ccd.util.CaseDataUtils.buildCaseData;
+import static uk.gov.hmcts.reform.sscs.ccd.util.CaseDataUtils.buildCaseDataMap;
+
+@Slf4j
+@ExtendWith(MockitoExtension.class)
+public class MultipleHearingsOutcomeMigrationTest {
```

Comment:
I don't think we need to retest everything about CaseOutcome migration, since the only method we're overriding in MultipleHearingsOutcomeMigration is getHearingsFromHmc. I think we should just test this method.